documentation/Operations.md

```diff
@@ -0,0 +1,92 @@
+# Operations
```

Comment:
Maybe this could be named e.g. `STANDALONE.md` for consistency with other filenames, and to better reflect the scope?

Reply:
I agree, we could also add more documentation about standalone here as well